source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -56,15 +37,15 @@ To get started, go to your Pantheon Dashboard and click **Download all Drush ali
 
 ###Install the Pantheon Drush Aliases
 
-If you are on Linux/MacOS environment, put the generated pantheon.aliases.drushrc.php in either the `.drush` directory in your home, or the `aliases` directory of your local Drush installation.
+If you are on Linux/Mac OS environment, put the generated pantheon.aliases.drushrc.php in either the `.drush` directory in your home, or the `aliases` directory of your local Drush installation. 
 
 Drush will search for aliases in any of these files using the alias search path. The following locations are examined for alias files:
 
 1. In any path set in `$options['alias-path']` in drushrc.php, or (equivalently) any path passed in via --alias-path=... on the command line.
 2. If 'alias-path' is not set, then in one of the default locations:
   - /etc/drush
-  - In the drush installation folder
-  - Inside the 'aliases' folder in the drush installation folder
+  - In the Drush installation folder
+  - Inside the 'aliases' folder in the Drush installation folder
   - $HOME/.drush 
 
 3. Inside the sites folder of any bootstrapped Drupal site, or any local Drupal site indicated by an alias used as a parameter to a command.

```

Comment:
1. 2. and 3. are documented in Drush; I don't think we need that much detail here. Wouldn't hurt to leave this here if you particularly want it, but I don't think we need it.